frontend/app/views/spree/shared/_image.html.erb

```diff
@@ -1,11 +1,11 @@
 <% size ||= :mini %>
 <% itemprop ||= nil %>
 
-<% if image && image.attachment? %>
+<% if image_url %>
   <% if itemprop %>
-    <%= image_tag image.attachment(size), itemprop: itemprop %>
+    <%= image_tag image_url, itemprop: itemprop %>
```

Comment:
Something to note here. We don't use an image's `alt` attribute in either frontend or backend. Would being able to access `image.alt` here be an argument for using a custom image class [like I suggest in this PR comment](https://github.com/solidusio/solidus/pull/2337/files#r154390915)? That image class could have a strict interface `#url(size)`, `#id` (for that single-use modal on `backend/image/_image_row`), `#viewable_id`, and `#alt`.

Is there a reason why we haven't historically been rendering an image's `alt` attribute? @mamhoff @tvdeyen

Reply:
> Would being able to access image.alt here be an argument for using a custom image class

Yes! A separate image class (`Picture` maybe?) makes total sense to abstract the presenter API 👍

Reply:
> Is there a reason why we haven't historically been rendering an image's alt attribute?

Not sure. We should render it (at least in the frontend).